primer/src/Primer/Action.hs

```diff
@@ -733,6 +741,58 @@ constructCase ze = do
         TC.SmartHoles -> flip replace ze <$> case_ (pure $ target ze) []
     _ -> throwError $ CustomFailure ConstructCase ("can only construct case on expression with type a exactly saturated ADT, not " <> show ty)
 
+addCaseBranch :: ActionM m => QualifiedText -> ExprZ -> m ExprZ
+addCaseBranch rawCon ze = case target ze of
```

Comment:
In general, all our "add/delete case branches" actions (both high and low level) are (will) be offered/accepted on the `case` node itself. This feels a bit odd but is I think the best we can do for now, since we don't have the ability to select one branch -- we can only select things in branches or binders in patterns. A different choice would be to give "add" at the root of the wildcard RHS and "delete" at the root of each non-wildcard branch's rhs